### ISSUE TITLE
Issue running php artisan voyager:install, error adding symlinks to public folder

### ISSUE THREAD

**Issue by stephenmorton** [thedevdojo/voyager#4416]
## Version information
 - Laravel: v2.1.0
 - Using latest version of Homestead on Windows 10
 - PHP: 7.3.8
 - Database: MySQL V8

## Description
When installing Voyager on a working laravel homestead dev setup, I get through fine to the step requiring php artisan voyager:install --with-dummy (ps, same issue, when not installing dummy data also)

Install gets to the line when Adding the storage symlink to your public folder

   ErrorException  : symlink(): Protocol error

  at /home/vagrant/code/voyager/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php:252
    248|      */
    249|     public function link($target, $link)
    250|     {
    251|         if (! windows_os()) {
  > 252|             return symlink($target, $link);
    253|         }
    254|
    255|         $mode = $this->isDirectory($target) ? 'J' : 'H';
    256|

Error seems to be installing this symlink.

Database is propogated fine.
When I log into the app/admin, I get an error in the header, complaining that the Image Symlink haven't been installed, and it gives me a button to click to install them, but I get a similar error.

There is also a command like command that also tried to do the same thing ( artisan storage:link ), however, this also gives a similar error.

Basically, it all seems to install correctly, with expection of this image/media symlink.

Any ideas what I have done wrong, or how to fix it?

Thanks in advance.

Per above, go through the install procedure, fail at step 4

## Expected behavior
Successful install without symlink error
![voyager-symlink](https://user-images.githubusercontent.com/11910775/64934086-cd6c6e80-d88b-11e9-8e4d-b1be43eba161.PNG)

Screenshot within the terminal is attached.

**Comment by fletch3555** [comment]
All the steps you've tried simply call `artisan storage:link` under the hood, so if you get the same error calling that command directly, then it's something with your laravel install or system setup.  I'm confident saying it's almost certainly nothing to do with Voyager.

**Comment by stephenmorton** [comment]
Hi fletch, that might be correct, but it is a bulk standard homestead  install,  using the instructions from the laravel site per the book. It would be good to know, what, if anything, I have messed up. I have several other apps, that I am working on in laravel on this VM that are working fine. I know that doesn't prove anything, but it is hard to see what I could have messed up with my homestead install to make this fail.

If it helps, here is a relevant snippit from my homestead.yaml file
<-- cut
folders:
    - map: C:\Users\sgm25\Documents\Development\Laravel\
      to: /home/vagrant/code

sites:
    - map: tt1.test
      to: /home/vagrant/code/blog/public

    - map: tt2.test
      to: /home/vagrant/code/project/public

    - map: voyager.test
      to: /home/vagrant/code/voyager/public
--> paste

This app is using the voyager.test map above.
In the .env file, I also made sure to reference the APP_URL like this  =  APP_URL=http://voyager.test

**Comment by stephenmorton** [comment]
Hi guys,

Anyone know where I should be looking to diagnose my issue. I am pretty sure it isn't an issue with voyager, and more with my setup, but does anyone have any pointers on where I should be looking or any tests I can try? Thanks in advance.

**Comment by MrCrayon** [comment]
Try with this https://stackoverflow.com/questions/39496598/laravel-5-3-storagelink-symlink-protocol-error
If that doesn't solve I'd suggest to open a question there.

**Comment by stephenmorton** [comment]
You are a genius MrCrayon, I recon you are spot on. Haven't tried it yet, but that will be the issue I recon. Thankyou.

**Comment by MrCrayon** [comment]
You are welcome.
Please remove the possible bug label, as we concluded is not Voyager related, and close the issue if you think it's solved.
